src/components/trace/hat-graph-node.ts

```diff
@@ -45,41 +54,31 @@ export class HatGraphNode extends LitElement {
             "
             line-caps="round"
           />
-          `
-      }
-    <g class="node">
-      <circle
-        cx="0"
-        cy="0"
-        r="${NODE_SIZE / 2}"
-      />
-      }
-      ${
-        this.badge
-          ? svg`
+          `}
+        <g class="node">
+          <circle cx="0" cy="0" r=${NODE_SIZE / 2} />
+          }
+          ${this.badge
+            ? svg`
         <g class="number">
           <circle
             cx="8"
-            cy="${-NODE_SIZE / 2}"
+            cy=${-NODE_SIZE / 2}
```

Comment:
Does this work? SVG is kind of picky about quotes.

Reply:
It will become a string, with quotes :-).